lib/recipe/refreshable_local_recipes.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Recipe
+  class RefreshableLocalRecipes
+    delegate_missing_to :local_recipes
```

Comment:
I really like this usage of delegation. It's a bit like the decorator pattern, but without having to inherit from SimpleDelegator or something similar ?